pxtsim/simdriver.ts

```diff
@@ -438,6 +441,7 @@ namespace pxsim {
             this.setState(starting ? SimulatorState.Starting : SimulatorState.Stopped);
             if (unload)
                 this.unload();
+            this.removeEventListeners();
```

Comment:
was this missing before?

Reply:
Yes. It seemed to be dead code. Will do further testing.

Reply:
Add was idempotent, but we never registered those events, so we were still listening from the home screen.